index.src.html

```diff
@@ -202,33 +202,20 @@
     };
 
     dictionary ClientData {
-        required DOMString       challenge;
-        required DOMString       facet;
-        required JsonWebKey      tokenBinding;
+        required DOMString           challenge;
+        required DOMString           facet;
         required AlgorithmIdentifier hashAlg;
-        WebAuthnExtensions          extensions;
+        JsonWebKey                   tokenBinding;
+        WebAuthnExtensions           extensions;
     };
 
     dictionary WebAuthnExtensions {
     };
 
-    interface AttestationStatement {
-        readonly    attribute AttestationHeader header;
-        readonly    attribute AttestationCore   core;
-        readonly    attribute DOMString         signature;
-    };
-
-    interface AttestationCore {
+    interface WebAuthnAttestation {
         readonly    attribute DOMString     type;
-        readonly    attribute unsigned long version;
-        readonly    attribute DOMString     rawData;
         readonly    attribute DOMString     clientData;
-    };
-
-    interface AttestationHeader {
-        readonly    attribute DOMString   claimedAAGUID;
-        readonly    attribute DOMString[] x5c;
-        readonly    attribute DOMString   alg;
+        readonly    attribute any           statement;
```

Comment:
As I mentioned in the commit message, I think that for Android attestation it would be preferable for this just to become "DOMString safetyNetResponse" and thereby eliminate a bunch of unnecessary manipulation of bits on the client side. That would allow us to delete an entire (sub)section of the spec with no loss in expressiveness.

So keeping this as any allows us to accommodate other attestation structures as needed.